package.json

```diff
@@ -25,7 +23,7 @@
     "stylelint": "^7.13.0"
   },
   "devDependencies": {
-    "@primer/css": "12.0.0-rc.481d796",
+    "@primer/css": "12.1.1",
```

Comment:
Should this be in `peerDependencies` or `dependencies`? Doesn't seem like it'll  be loaded here if someone installs `@primer/stylelint-selector-no-utility`